code/game/objects/items/stacks/stack.dm

```diff
@@ -401,6 +401,7 @@
 
 	if(created)
 		created.setDir(builder.dir)
+		on_item_crafted(builder, created)
```

Comment:
All cool and dandy,  but this means only crafted items are affected by this, not ones that are pre-mapped or spawned through other means.

Reply:
I'm okay with this